messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
 
-Invariants
-^^^^^^^^^^
+Preconditions
+^^^^^^^^^^^^^
 
-Only valid if all the following hold:
+For a ``LockedTransfer`` to be considered valid the following conditions have to be true.
 
 - There is a channel which matches the given :term:`chain id`, :term:`token network` address, and :term:`channel identifier`.
 - The corresponding channel is in the open state.
 - The :term:`nonce` is increased by ``1`` in respect to the previous :term:`balance proof`
 - The :term:`locksroot` must change, the new value must be equal to the root of a new tree, which has all the previous locks plus the lock provided in the message.
```

Comment:
- we don't have a tree anymore, right? It is more like a hash-list or so. There is certainly a specific term for that

Reply:
I am under the impression that it was removed, but @hackaugusto @pirapira can provide more information about that.

Reply:
"the root of a new tree" should be "the hash of a new list of pending locks".  Sorry I haven't changed this (I was only searching for the word "Merkle").